amx/server/syscalls.lua

```diff
@@ -626,7 +628,7 @@ g_SAMPSyscallPrototypes = {
 	SetTrainDirection = {'v', 'b'},
 	GetTrainSpeed = {'v', 'r'},
 	SetTrainSpeed = {'v', 'f'},
-	GetVehicleDriver = {'v'},

```

Comment:
Where's `GetVehicleDriver` now?

Reply:
It's actually not a SA-MP server native, it was added in open.mp server (along with GetVehicleOccupant, but they have their own reasons to keep them both, TLDR their milestone was to build-in some natives from popular plugins). I added GetVehicleDriver recently in one of previous PRs, and since it wasn't here for years before, I thought it might still be tweaked without deprecations or aliases to a less appropriate function name 😄

Reply:
My apologies, I will mark this as resolved now. I had this declared in some section of my script 😬